### ISSUE TITLE
Seeing same behavior described in #1242 in 0.13.3 when VIRTUAL_ENV is set.

### ISSUE THREAD

**Issue by naunga** [davidhalter/jedi#1332]
After a few minutes of working in Atom with the ide-python package installed, I start to run out of file descriptors. 

I'm using pipenv to manage my virtual environment. 

My environment looks like this:
- macOS 10.14.4
- Python 3.7.3
- python-language-server 0.26.1
- jedi 0.13.3
- Atom 1.37.0

Confirmed VIRTUAL_ENV is set:
![screenshot](https://user-images.githubusercontent.com/2334258/58262531-707f0280-7d48-11e9-8539-485049c90b52.png)

Not sure if this is maybe an issue with pyls, but figured I'd let you all know.

**Comment by davidhalter** [comment]
That sounds strange. No idea if maybe you are using a "different" Jedi than the one you're listing.

Can you try master? There were a few fixes specifically to MacOS.

**Comment by naunga** [comment]
I can certainly give master a try. Late last night I found that if I specify the _exact_ path to the python executable my virtual environment is using in Atom, then this behavior stops. 

It was using the correct python executable if I just specified `python`, but I think there may be some branching logic in the ide-python extension that handles a fully qualified path differently.

I should have some time this weekend to dig into this more.
